### DISCUSSION TITLE
VK_ERROR_LAYER_NOT_PRESENT 是因为我没有安装 vulkan 驱动吗？

### DISCUSSION THREAD

**Discussion by car520120** [BoomingTech/Piccolo#343] [General]
![image](https://user-images.githubusercontent.com/36702004/186881291-bfdf628b-de4b-41f5-b69e-de84731a44a4.png)
VK_ERROR_LAYER_NOT_PRESENT

**Comment by kwbm** [comment]
可以提供一下显卡信息、驱动版本吗

**Comment by car520120** [comment]
![image](https://user-images.githubusercontent.com/36702004/189008912-65a38fb9-9590-493b-adff-5a062adec67e.png)
![image](https://user-images.githubusercontent.com/36702004/189010205-3da18aa6-a496-4e73-85a6-85f85fed290c.png)
显卡是 750ti 来的，我跟 godot 也是 vulkan  来的能正常跑

**Comment by kwbm** [comment]
可以试一下安装vulkan sdk和升级一下显卡驱动，在开始菜单中有一个vulkan configurator，打开看一下支持情况呢？
![image](https://user-images.githubusercontent.com/102910208/189016562-ee1e07bc-1db6-425e-84ef-a068fba3d4b3.png)

![image](https://user-images.githubusercontent.com/102910208/189016442-c881babe-acd4-488a-8e0b-a1d39556c509.png)
